cxmon/src/mon_cmd.cpp

```diff
@@ -303,6 +307,288 @@ void binary_dump(void)
 
 
 /*
+ * Add Break Point
+ */
+void break_point_add(void)
+{
+	uintptr address;
+
+	if (mon_token == T_END ||
+			!mon_expression(&address)) {
```

Comment:
Nit: Don't line break before this since it's short enough.

Reply:
Done!